src/patterns/components/inputs/collection.yaml

```diff
@@ -89,6 +89,12 @@ variableTable:
   $sprk-text-input-border-radius:
     default: 4px
     description: The border radius to use for text input elements.
+  $sprk-text-input-huge-border:
+    default: 1px solid $sprk-black-tint-25
+    description: The border around huge text input elements.
+  $sprk-text-input-huge-focus-border-width:
```

Comment:
Does sprk-b-Label--huge need to be removed too?

Reply:
yup it's been removed!